backend/src/ApiRequest/Queries/UpdateDatabase.graphql

```diff
@@ -1,5 +1,5 @@
 mutation UpdateDatabase(
-    $input: UpdateDatabaseInput!
+    $input: DatabaseRequestDto!
```

Comment:
I personally prefer the old naming here. DTO is very general as a name. The suffix `Input` conferred more information about the intended usage of the class and with the suffix `Input` the class name also matches the type name of the GraphQL schema in which the `UpdateDatabase` mutation is defined.